src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,51 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    public static final String DATA_SEPARATOR = " ";
+    public static final String COMMA = ",";
```

Comment:
```suggestion
    public static final String CSV_SEPARATOR = ",";
```
we should call variables according to their purpose and not the value they hold